src/base/base-mixin.js

```diff
@@ -668,6 +715,28 @@ export class BaseMixin {
         return result;
     }
 
+    _makeKeyboardAccessible (onClickFunction, ...onClickArgs) {
+        // called from each chart module's render and redraw methods
+        const tabElements = this._svg
+            .selectAll('.dc-tabbable')
+            .attr('tabindex', 0);
+                
+        if (onClickFunction) {
+            tabElements.on('keydown', (d, i) => {
```

Comment:
Since you started this PR, D3 was updated to 6.0 and the signature of event handlers [changed](https://github.com/d3/d3/blob/master/CHANGES.md#changes-in-d3-60) - now the signature is `(event, d)`

We built an adapter for dc@4.1 called `adaptHandler` that smooths over the difference, to move to d3@6 but still support d3@5. I would suggest using it here.